apps/admin/frontend/src/components/smartcard_modal/smartcard_modal.test.tsx

```diff
@@ -367,9 +367,9 @@ test('Programming election manager and poll worker smartcards', async () => {
     await waitFor(() =>
       expect(screen.queryByRole('alertdialog')).not.toBeInTheDocument()
     );
-    // For some reason, finding by role doesn't work here, though 'Election Definition' is present
-    // in a heading
-    await screen.findByText('Election Definition');
+    // For some reason, we need to use the "hidden" option here, even though the
+    // heading is not hidden
+    await screen.findByRole('heading', { name: 'Election', hidden: true });
```

Comment:
My guess is that this is a `react-modal` issue similar to the one that prompted @kofi-q to add `hackActuallyCleanUpReactModal` to some tests.